tests/integration/features/steps/integration_steps.py

```diff
@@ -479,11 +478,11 @@ def get_args(context, storage_provider, client_encryption, cassandra_url, use_mg
         )
 
     grpc_args = {
-        "enabled": grpc
+        "grpc_enabled": grpc
     }
 
     kubernetes_args = {
-        "enabled": use_mgmt_api,
+        "k8s_enabled": use_mgmt_api,
```

Comment:
Issue:
Here as well, I don't see where `k8s_enabled` is used. It looks like the field checked in config.py is `enabled` still.

Reply:
From here, the kubernetes_args get flattened into the `args` but then it probably **does not** get properly used [here](https://github.com/thelastpickle/cassandra-medusa/blob/master/medusa/config.py#L208-L218). I'll go ahead and try to add an unit test for this.

Reply:
Awesome. Glad my questions actually amounted to more than just you educating me!